cas/src/main/resources/templates/fragments/authProviders.html

```diff
@@ -2,20 +2,27 @@
 <html lang="en" xmlns:th="http://www.thymeleaf.org">
     <body>
 
-        <th:block th:fragment="URS (loginUrl, autoRedirect)">
+        <th:block th:fragment="maapauth (loginUrl, autoRedirect)">
             <a th:href="@{${loginUrl}}" th:attr="autoredirect=${autoRedirect}" th:title="'Login with EarthData Account'" class="auth-provider-card">
                 <img src="images/urs-logo.png" th:alt-title="'EarthData Logo'" />
                 <span class="btn">Login with EarthData Account</span>
             </a>
         </th:block>
 
 
-        <th:block th:fragment="ESA (loginUrl, autoRedirect)">
+        <th:block th:fragment="esa (loginUrl, autoRedirect)">
             <a th:href="@{${loginUrl}}" th:attr="autoredirect=${autoRedirect}" th:title="'Login with ESA Account'" class="auth-provider-card">
                 <img src="images/esa-logo.png" th:alt-title="'European Space Agency (ESA) Logo'" />
                 <span class="btn">Login with ESA Account</span>
             </a>
         </th:block>
+        
```

Comment:
We support Google Logins now?

Reply:
This was more for testing purposes to help with upgrades, so I'll remove it.